Loop/Managers/LoopDataManager.swift

```diff
@@ -912,6 +1101,20 @@ extension LoopDataManager {
         )
     }
 
+    private func computeRetrospectiveGlucoseEffect(startingAt glucose: GlucoseValue, carbEffects: [GlucoseEffect]) -> [GlucoseEffect] {
+        let retrospectiveGlucoseDiscrepancies = insulinCounteractionEffects.subtracting(carbEffects, withUniformInterval: carbStore.delta)
+        let retrospectiveGlucoseDiscrepanciesSummed = retrospectiveGlucoseDiscrepancies.combinedSums(of: settings.retrospectiveCorrectionGroupingInterval * 1.01)
```

Comment:
[nit] `1.01` feels like a magic number without explanation.

Reply:
Copy/pasted from elsewhere in the file. Pulled it out into a named constant.